tipocket-ctl/tpctl/scripts/env_raw.sh

```diff
@@ -0,0 +1,72 @@
+# WARNING: All commands connect to `tidb` container of tidb pods by default.
+# 	   As tidb pod has two containers
+OUTPUT_DIR=output
+CASE_POD_ID=$(argo get -n argo $DEPLOY_ID -o json | jq -r '.status.nodes[] | select(.type == "Pod" and .templateName != "notify") | .id')
+CLUSTER_NAMESPACE=$(argo get -n argo $DEPLOY_ID -o json | jq -r '.spec.templates[-1].container.command[2]' | grep -oP '\-namespace=".*?"' | grep -o '".*"' | sed -e 's/^"//' -e 's/"$//')
```

Comment:
Can we shorten this line?

Reply:
Shorten it may make it harder to understand, especially for someone who don't familar with shell tools.
I've considered to finish the extraction of --namespace="" in a single sed line, but it would make this hard to read.

Reply:
How about using `\`?

```
CLUSTER_NAMESPACE=$(argo get -n argo $DEPLOY_ID -o json | \
   jq -r '.spec.templates[-1].container.command[2]' | \
   grep -oP '\-namespace=".*?"' | \
   grep -o '".*"' | \
   sed -e 's/^"//' -e 's/"$//')
 

```

Reply:
Good